tests/rules/test_quoted_strings.py

```diff
@@ -14,7 +14,6 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 from tests.common import RuleTestCase
-
```

Comment:
Is there a config to keep this blank line? `tests` are standard, while `yamllint` is third-party.

Reply:
I think it might classify `tests` as first-party just like `yamllint`, because it discovers the `tests` directory. Let me try these:
```
[lint.isort]
extra-standard-library = ["tests"]
```
```
[lint.isort]
known-first-party = ["yamllint"]
```

Reply:
I pushed this setting:
```toml
[lint.isort]
known-first-party = ["yamllint"]
```
Do you like it?

Reply:
Wrong one. I thought it worked but it doesn't.

Reply:
This one works slightly better:
```toml
[lint.isort]
extra-standard-library = ["tests"]
```
That's still not what we want, because tests are definitely **not** standard. Rather they are another category of first-party, or maybe third-party.

Reply:
Therefore I think the closest to what you want is:
```toml
[lint.isort]
known-third-party = ["tests"]
```